README.md

```diff
@@ -194,6 +194,23 @@ following rules:
 For more details about authorization in LoopBack 4, refer to
 https://loopback.io/doc/en/lb4/Loopback-component-authorization.html.
 
+### Tests
+
+This repository comes with integration, unit, acceptance and end-to-end (e2e)
+tests. To execute these, see instructions below.
+
+Do the following to execute e2e tests:
```

Comment:
Can you also add a note for them to make sure the server is running?

Reply:
Added